coc_reporting.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: default
+title: "Code of Conduct: Reporting Guide"
+---
+
+Code of Conduct: Reporting Guide
+--------------------------------
+
+Largely taken from [django](https://www.djangoproject.com/conduct/reporting/). Thanks!
+
+If you believe someone is violating the code of conduct we ask that you report it to the Dev Edmonton Society Board by emailing conduct AT devedmonton DOT org. **All reports will be kept confidential.** In some cases we may determine that a public statement will need to be made. If that's the case, the identities of all victims and reporters will remain confidential unless those individuals instruct us otherwise.
+
```

Comment:
What are our views on anonymous reports?  Having looked at this in the past it's harder than you'd think to create a "dropbox" that's actually anonymous.  In particular if you want to have a further conversation with the reporter for clarification, etc. to have an anonymous platform that allows for 2-way conversation is more involved.  Not sure if it's worth worrying about, but it always makes me a little uncomfortable to require people to identify themselves in order to report.

Reply:
@Estyn had some questions about this too. Here's my take...

If the mission of our code of conduct and the reporting process is to identify and address issues that undermine the culture and diversity of our community, then there are some pros and cons to anonymous reporting. 

### Pros
* Ensure that people don't hesitate to report issues
* Ensures that reporters identities aren't released by us (as we don't know them!)

### Cons
* Without being able to identify individuals, the scope and volume of issues is unknown
* Truly "anonymous" reporting is very hard to setup
* To act on a situation, and ensure it's resolved, we may require more information and context which anonymous reports don't provide
* There may be speculation about the source of anonymous reporters
* Though unlikely, anonymous reporting allows bad actors to disrupt the community

By building a firewall within our organization, we can ensure that only parties directly involved in a report are privy to the details and identities of those involved. We *need* to convince people in our community that we take reporter privacy seriously, and understand the personal and professional risks they may be taking.

I think we could address this a bit better by indicating that Dev Edmonton won't ever share reporter identities with the public, or with the individual a grievance is against. This includes restricting access to information that could be used to identify a reporter (like dates/times, names, phone numbers, emails, aliases, etc.) That said we will share the report with a subset of the board required to address it properly.

I'll try to update the language in the reporting page and CoC to make this clearer.